presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -64,22 +65,25 @@ def Apply(self, request, context):
         return response
 
     def get_language_from_request(self, request):
-        language = request.analyzeTemplate.languageCode
+        language = request.analyzeTemplate.language
         if language is None or language == "":
             language = DEFAULT_LANGUAGE
         return language
 
-    def analyze(self, text, entities, language):
+    def analyze(self, text, entities, language, all_fields=False):
```

Comment:
default 'False' is not needed as the only place that calls this method is lie 57, which passes a value.
you probably need to update some tests after your remove the default value